src/app/projects/projects.page.test.tsx

```diff
@@ -0,0 +1,11 @@
+import Projects from '@/app/projects/page';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+describe('Footer', () => {
```

Comment:
Great work, please use `Projects` instead of `Footer` in the test suite name

Reply:
lol awkward oversight